### PR TITLE
[Feature][Doc] Add docs for zeppelin task plugin

### LINKED ISSUE #9838: [Feature][Doc] Add docs for zeppelin task plugin
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

* We need to add docs for `zeppelin task plugin`.
* Besides traditional docs like screenshots or descriptions to instruct users how to use `zeppelin task plugin`, we could also add an example `docker-compose` file to help users get their hands dirty more easily.
* related: #9201 

### Use case

* Already described above.

### Related issues

* Already described above.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by EricGao888**
## Purpose of the pull request
* Add docs for zeppelin task plugin.
* This PR closes: #9838 

## Brief change log
* Add docs for zeppelin task plugin.

### CODE REVIEW

**Review by caishunfeng — COMMENTED** [review]
LGTM.
PTAL @zhongjiajie @Tianqi-Dotes

File: docs/docs/en/guide/task/zeppelin.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+This example illustrates how to create a zeppelin paragraph task node.
+
+![demo-zeppelin-paragraph](/img/tasks/demo/zeppelin.png)
+
+![demo-get-zeppelin-id](/img/tasks/demo/zeppelin_id.png)
+
+
+
+
```
Comment (zhongjiajie) [inline docs/docs/en/guide/task/zeppelin.md]:
```suggestion

```

**Review by zhongjiajie — DISMISSED** [review]

**Comment by zhongjiajie** [comment]
@EricGao888 Should also change the dev config and add zeppelin task in https://github.com/apache/dolphinscheduler/blob/dev/docs/configs/docsdev.js#L168

**Comment by EricGao888** [comment]
> @EricGao888 Should also change the dev config and add zeppelin task in https://github.com/apache/dolphinscheduler/blob/dev/docs/configs/docsdev.js#L168

@zhongjiajie Fixed in the latest commit. Thx!

File: docs/docs/en/guide/task/zeppelin.md (hunk 1 above)
Comment (zhongjiajie) [inline docs/docs/en/guide/task/zeppelin.md]:
```suggestion
```

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Comment by zhongjiajie** [comment]
@EricGao888  tips, you could ignore and do not add issue number in PR title.

**Comment by EricGao888** [comment]
> @EricGao888 tips, you could ignore and do not add issue number in PR title.

@zhongjiajie Sure, thx for the reminder~

**Pull request merged**